spec/lib/api/v3/projects/schemas/project_schema_representer_spec.rb

```diff
@@ -102,6 +102,73 @@
       end
     end
 
+    describe "_attributeGroups" do
+      context "without existing sections" do
+        it "renders an empty section attribute group" do
+          expect(subject)
+            .to be_json_eql([].to_json)
+            .at_path("_attributeGroups")
+        end
+      end
+
+      context "with existing sections" do
+        let!(:user_section) { create(:project_custom_field_section, name: "Common Project Attributes") }
+        let!(:user_custom_field) do
+          create(:text_project_custom_field, project_custom_field_section: user_section)
+        end
+        let!(:admin_section) { create(:project_custom_field_section, name: "Admin Project Attributes") }
+        let!(:admin_custom_field) do
+          create(:text_project_custom_field, project_custom_field_section: admin_section, admin_only: true)
+        end
+
+        before do
+          allow(contract.model)
+            .to receive(:available_custom_fields)
+            .and_return(ProjectCustomField.all)
+        end
+
+        context "with a user" do
+          it "renders section attribute group elements of the schema" do
+            expect(subject)
+              .to be_json_eql(
+                {
+                  _type: "ProjectFormCustomFieldSection",
+                  name: "Common Project Attributes",
+                  attributes: ["customField#{user_custom_field.id}"]
+                }.to_json
+              )
+              .at_path("_attributeGroups/0")
```

Comment:
I think better to check complete array:
```suggestion
              .to be_json_eql(
                [
                  {
                    _type: "ProjectFormCustomFieldSection",
                    name: "Common Project Attributes",
                    attributes: ["customField#{user_custom_field.id}"]
                  }
                ].to_json
              )
              .at_path("_attributeGroups")
```